sewer/dns_providers/tests/test_route53.py

```diff
@@ -0,0 +1,124 @@
+import mock
+import json
+from unittest import TestCase
+from botocore.stub import Stubber
+
+import sewer
+
+
+class TestRoute53(TestCase):
+    """
+    """
+
+    def setUp(self):
+        self.domain_name = "example.com"
+        self.domain_dns_value = "mock-domain_dns_value"
+        self.route53_key_id = "mock-key-id"
+        self.route53_key_secret = "mock-key-secret"
+        self.dns_class = sewer.Route53Dns(self.route53_key_id, self.route53_key_secret)
+
+    def tearDown(self):
+        pass
+
+    @staticmethod
+    def mocked_route53_set_record_response():
+        return {"ChangeInfo": {"Id": "mocked-id"}}
+
+    def make_change_batch(self, action, domain_name, domain_value):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/app/komuW/sewer/pullRequest?prid=4612436)